src/main/docker/mailhog.yml

```diff
@@ -0,0 +1,27 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# Mailhog service
```

Comment:
```suggestion
# Mailhog service
#
# Necessary changes to your Artemis config to use this container as email service:
# ```
# spring:
#     mail:
#          host: mailhog
#          port: 1025
#          username:
#          password:
# ```
```
I’d say specifying it here should be fine. Most developers won’t need the email service set up. If some need it they directly find the necessary changes here, rather than in an env-file somewhere else.